crnk-example-service/src/main/java/io/crnk/example/service/model/MovieEntity.java

```diff
@@ -58,4 +60,35 @@
 	@Version
 	private Integer version;
 
+    @Transient
+    @JsonApiLinksInformation
+    private MovieLinks links = new MovieLinks();
+
+    public static class MovieLinks implements  LinksInformation
+    {
+   
+    	private String rates;
+    	private String comments;
+    
+		public String getRates() {
+			
+			return "https://www.imdb.com/" + getId() + "/rates";
```

Comment:
@remmeier please check the updates I added the self link also but is there a way so the self link can return the base path also beside the endpoint  like before customizing the links 
before customizing links 
it will return "localhost:8080/movie/{id}"
now I overrided it to return /movie/{id}